### ISSUE TITLE
nw.js 0.13.0-rc1 - Window.open(url, [options], [callback])  - window in callback undefined

### ISSUE THREAD

**Issue by finim** [nwjs/nw.js#4512]
System: Windows 8.1 x64
nw.js: 0.13.0-rc1 x64 sdk build

When calling Window.open(url, [options], [callback]), the window instance put into the callback is often undefined. Example:
```javascript
var local = {};
local.gui = require('nw.gui');

local.gui.Window.open('https://www.github.com', {
                    position: 'center',
                    width: 1410,
                    height: 700,                 
                    show: true
                }, function (_window) {
                    if (_window) {
                        console.log('OK. new window instance is there');
                    } else {
                        console.log('ERROR. new window instance is NOT there!');                  
                        console.log(typeof _window)
                    }
   });
```

It happens very often that _window is "undefined" in the callback. 
The window seems to be created but not shown and the instance is not given into the callback function. This does not happen every time but very often.

**Comment by f0rmiga** [comment]
same here using Mac OSX

**Comment by Gimmeaphatbeat** [comment]
+1

Windows 8.1

**Comment by tolrtd** [comment]
Might be the same problem as this #4418

**Comment by rogerwang** [comment]
Could you please try again with the latest nightly build?  http://dl.nwjs.io/live-build/04-19-2016/f0a0ece-fda17ad-6f7e880-c56a187/v0.14.2/

It tried the sample many times and it never fail.

btw, we also have the test case to guard this usage:  https://github.com/nwjs/nw.js/tree/nw14/test/remoting/open-remote

**Comment by rogerwang** [comment]
This was fixed in 0.14.4

**Comment by AndryBray** [comment]
https://github.com/nwjs/nw.js/issues/4844#issuecomment-218990338
